Tutorials/Model_Agnostic_Analysis.ipynb

```diff
@@ -0,0 +1,3281 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #1.    SST_time_mean = SST.mean(SST.cf['time'].name)
</pre><p><code>SST.cf.mean("time")</code>?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/COSIMA/cosima-recipes/pull/226/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='28'/>

Reply:
Awesome. Did not not know about that. Thanks. Have incorporated in notebook, looks much cleaner.